### ISSUE TITLE
ROI

### ISSUE THREAD

**Issue by Epta13** [guiwitz/napari-sediment#8]
The default main and the small ROIs have wierd size - thin and wide, probably wrong coordinates or so.

**Comment by guiwitz** [comment]
The main roi should encompass the largest possible region given the current mask. It can be that there's something wrong with your mask as there seems to be an issue with it as well in #9. Maybe you can try to make a very simple mask (annotation a few places manually) to check that thing work?

**Comment by Epta13** [comment]
Yes, I think it's the border mask which is not generated correctly. Once the mask is correct, the ROI works fine.
One small adjustment would be to automatically adjust th length of the samll ROI to the main ROI. E.g., I create the main ROI, adjust the size as I need and then I wanna create smal ROI, but now it creates them in the same extend as the original main ROI. Would it be possible to read the main ROI size for creating the smal ROI?

**Comment by guiwitz** [comment]
This is actually already the expected behaviour so I'm not sure why you don't see it. For example in this example, the main roi discards some top and bottom rows, and when I add a small roi it fits inside the main roi. I'll have to investigate live.
![image](https://github.com/guiwitz/napari-sediment/assets/4622767/08cd5562-e331-4fd2-9afb-7a245ff33dc8)

**Comment by Epta13** [comment]
Is it possible that the small ROI will always take the height of the main ROI instead of the border mask?

**Comment by guiwitz** [comment]
This is fixed now by requiring the user to first crop the image with main before adding an analysis ROI.
